### DISCUSSION TITLE
Demo env

### DISCUSSION THREAD

**Discussion by Mister-Hope** [ArtalkJS/Artalk#365] [Q&A] [answered]
Hi, this is Mr.Hope from @vuepress team, as users are requesting artalk support at https://github.com/vuepress-theme-hope/vuepress-theme-hope/issues/2733, I would like to support it in `vuepress-plugin-comment2` plugin.

My problem is that the comment panel on official docs does not work well and return 514 err code, and I probably do not want to deploy a demo env for this at my own server as I have production instances on them.

My concern is that can your team managed to provide a stable demo server which could be used in your documatation as well as 3rd party demo sites? Or are there any free services which I can deploy a demo instance with artalk for plugin users to give a try?

**Comment by qwqcode** [comment, marked as answer]
Dear Mr.Hope,

Thank you for reaching out to us. We appreciate your support.

We would like to inform you that the official documentation http response error 514 is a result of our current deployment of the Artalk service, which is currently limited to mainland China. We are working to expand our support to other regions in the near future, and we apologize for any inconvenience this may have caused you.

If you would like to continue using Artalk, you may try cloning the project and running a local development instance by following the instructions in the CONTRIBUTING.md file on our GitHub repository: https://github.com/ArtalkJS/Artalk/blob/master/CONTRIBUTING.md.

Please note that this local development instance should not be used on your production server. If you have any questions or need assistance, we will be happy to help.

Thank you for your understanding and continued support.

Best regards,
Artalk Team
